topics/RDMA.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
+Valkey module that can be dynamically loaded on demand.
+
+## Getting Started
+
+[RDMA](https://en.wikipedia.org/wiki/Remote_direct_memory_access)
+enables direct data exchange between networked computers' main memory,
+bypassing processors and operating systems.
+
+As a result, RDMA offers better performance compared to TCP/IP. Test results indicate that
+Valkey Over RDMA achieves approximately 2 times higher QPS and lower latency.
+
+Please note that Valkey Over RDMA is currently supported only on Linux.
+
+## Running manually
+
+To run a Valkey server with RDMA mode:
+
+    ~# ./src/valkey-server --protected-mode no \
+         --loadmodule src/valkey-rdma.so bind=192.168.122.100 port=6379
+
+Bind address/port of RDMA can be modified at runtime using the following command:
+
+    192.168.122.100:6379> CONFIG SET rdma-port 6380
+
+Valkey can run both RDMA and TCP/IP concurrently on the same port:
+
+    ~# ./src/valkey-server --protected-mode no \
+         --loadmodule src/valkey-rdma.so bind=192.168.122.100 port=6379 \
+         --port 6379
+
+Or append `loadmodule src/valkey-rdma.so bind=192.168.122.100 port=6379` in valkey.conf, then:
+
+    ~# ./src/valkey-server valkey.conf
+
+Note that the network interface (192.168.122.100 of this example) should support
```

Comment:
I just noticed this.  It is a helpful hint to a naive user.  Perhaps there should be a "Prerequisites" or "Dependencies" section.